.teamcity/settings.kts

```diff
@@ -0,0 +1,76 @@
+import jetbrains.buildServer.configs.kotlin.v2019_2.*
+import jetbrains.buildServer.configs.kotlin.v2019_2.buildSteps.script
+import jetbrains.buildServer.configs.kotlin.v2019_2.buildSteps.powerShell
+import jetbrains.buildServer.configs.kotlin.v2019_2.buildFeatures.pullRequests
+import jetbrains.buildServer.configs.kotlin.v2019_2.buildFeatures.xmlReport
+import jetbrains.buildServer.configs.kotlin.v2019_2.buildFeatures.XmlReport
+import jetbrains.buildServer.configs.kotlin.v2019_2.buildSteps.nuGetPublish
+import jetbrains.buildServer.configs.kotlin.v2019_2.triggers.vcs
+import jetbrains.buildServer.configs.kotlin.v2019_2.vcs.GitVcsRoot
+
+version = "2021.2"
```

Comment:
Josh or James could correct me if I'm wrong, but my understanding was that we don't need the version specified here anymore...

Reply:
@JPRuskin @Windos for visibility.

Reply:
We also have the version specified on the Chocolatey/choco repository builds. I'm going to resolve this conversation as the build will work with it, and we can address cleanup and/or standardization of build configurations at a later date.